src/control/zoom/ZoomControl.cpp

```diff
@@ -113,6 +142,11 @@ void ZoomControl::endZoomSequence() {
     zoomSequenceStart = -1;
 }
 
+void ZoomControl::cancelZoomSequence() {
```

Comment:
Please fence this against `zoomSequenceStart == -1` to avoid entering an invalid state should this ever be called with no current zoom sequence.

Reply:
Done